### DISCUSSION TITLE
Stattrak & Seeds, Patterns in SkinChanger

### DISCUSSION THREAD

**Discussion by mdng568** [danielkrupinski/Osiris#3606] [Q&A]
Can u add Stattrak & Seeds, Patterns in SkinChanger?

I think from my point of view adding StatTrak, Pattern... will cause bugs or mb crack. But can you somehow add it?

Thank you very much. Thanks Osiris!

**Comment by Metaphysical1** [comment]
crack?
just change it from config file
